internal/template/config.go

```diff
@@ -67,6 +67,36 @@ func NewConfig(tplSource, data *merge.Merger, excluded []string) (Config, error)
 	return cfg, nil
 }
 
+func NewConfigWithoutData(tplSource *merge.Merger, excluded []string) (Config, error) {
+	var cfg Config
+
+	if *tplSource.GetCustom() == nil {
```

Comment:
this is a potential null pointer exception, chack for tplSource to be not nil
```suggestion
	if tplSource == nil || *tplSource.GetCustom() == nil {
```